### ISSUE TITLE
Cooked Diced Potatoes with Beef doesn't return a Bowl?

### ISSUE THREAD

**Issue by Burgersim** [axperty/moredelight-fabric#11]
I'm playing on 1.20.1 and just noticed that this doesn't seem to return a wooden bowl. Is that intentional or an oversight? Other stuff from Farmer's delight does return Bowls. Maybe this also happens with other food from this mod? Haven't been able to try yet though.

**Comment by axperty** [comment]
Hi Burgersim,

I'll take a look and I'll update you on this, it might be a bug because it's supposed to drop you a bowl after you eat cooked diced potatoes.

Thank you for playing with More Delight.

Have a good day,
Axperty.

**Comment by axperty** [comment]
Hi Burgersim,

I tested this and found that Cooked Diced Potatoes with Beef doesn't give a bowl when eaten in Fabric 1.20.1. Although I investigated further and the code seems to instruct the game to return the bowl, this behavior differs from the Forge version where you still receive a bowl. This might be an issue with More Delight or most likely with Fabric itself. I also tried replicating this in Expanded Delight (the mod More Delight is based in), and the same thing happens.

This behavior might change in a future update when More Delights adds support for Minecraft Fabric 1.21, but don't worry, it will keep receiving updates for Minecraft Fabric 1.20.1.

If you have any more questions, suggestions, or other bug reports, feel free to [submit a new issue.](https://github.com/axperty/moredelight-fabric/issues/new)

Have a good day,
Axperty.
